docs/cellxgene_census_schema.md

```diff
@@ -872,8 +881,185 @@ Cell metadata MUST be encoded as a `SOMADataFrame` with the following columns:
+
+`scene_id` MUST correspond to the values `scene_id` in `census_obj["census_spatial_sequencing"][organism].obs_presence_matrix`.
+
+For each observation in each Scene, spatial array positions and additional positional metadata MUST be encoded as a `PointCloudDataFrame`.  Each row corresponds to an observation with the following columns:
+
+<!-- markdownlint-disable reference-links-images -->
+<table>
+<thead>
+  <tr>
+    <th>Column</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>x</td>
+    <td>float</td>
+    <td>It MUST be the corresponding value in the <b>first</b> column of <code>obsm["spatial"]</code>. As defined in the CELLxGENE dataset schema.</td>
+  </tr>
+  </tr>
+    <td>y</td>
+    <td>float</td>
+    <td>It MUST be the corresponding value in the <b>second</b> column of <code>obsm["spatial"]</code>. As defined in the CELLxGENE dataset schema.</td>
+  </tr>
+  <tr>
+    <td>soma_joinid</td>
+    <td>integer</td>
+    <td>It MUST be valid <code>soma_joinid</code> from <code>census_obj["census_spatial_sequencing"][organism].obs</code>.</td>
+  </tr>
+  </tr>
+  </tr>
+ </tbody>
+</table>
+<!-- markdownlint-enable reference-links-images -->
+
+If Visium ("EFO:0010961") the units for the spatial array positions are pixels from the high-resolution image (`spatial[scene_soma_joinid].img[library_id]`). Otherwise TBD.
+
+The location dataframe MUST have the metadata fields:
+
+<!-- markdownlint-disable reference-links-images -->
+<table>
+<thead>
+  <tr>
+    <th>Key</th>
+    <th>Encoding</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>soma_geometry_type</td>
+    <td>str</td>
+    <td>MUST be "radius"</td>
+  </tr>
+  </tr>
+    <td>soma_geometry</td>
+    <td>float</td>
+    <td>Radius of points: <code>dimeter/2</code>. If Visium ("EFO:0010961") <code>diameter</code> MUST be <code>.uns["spatial"][library_id]['spot_diameter_fullres']</code>.  As defined in the CELLxGENE dataset schema. Otherwise TBD-TODO (else for Slide-seq it should be  0.003% of the radius occupied by the full cloud of points).</td>
+  </tr>
+ </tbody>
+</table>
+<!-- markdownlint-enable reference-links-images -->
+
+#### Images of a Scene - `census_obj["census_spatial_sequencing"][organism].spatial[scene_soma_joinid].img[library_id]` – `Collection` of `MultiscaleImage`
+
+Images of a Visium ("EFO:0010961") scene MUST adhere to the following specifications. Other assays MUST NOT have images, and MUST NOT include the `img` collection.
+
+`library_id` MUST be the corresponding value in the source H5AD slot `.uns["spatial"][library_id]`, as defined in the CELLxGENE dataset schema.
+
+##### High resolution image of a Scene  – `census_obj["census_spatial_sequencing"][organism].spatial[scene_soma_joinid].img[library_id]["highres_image"]` – `MultiscaleImage`
+
+The high resolution image of a Visium ("EFO:0010961") scene MUST be included and MUST be encoded as a `SOMAImageNDArray`.
```

Comment:
If you're going to use this style - Visium ("EFO:0010961") - to reference to assays, then I'd recommend using the correct label - Visium Spatial Gene Expression.

Reply:
Just to clarify, would you like me to change `Visium ("EFO:0010961")` to `Visium Spatial Gene Expression` or `Visium Spatial Gene Expression ("EFO:0010961")`?

Reply:
Visium Spatial Gene Expression ("EFO:0010961") or "EFO:0010961" for _Visium Spatial Gene Expression_